Update Get Started Guide structure (#11875)
openvinotoolkit/openvino: docs/install_guides/pypi-openvino-dev.md

### Step 4. Install the Package

There are two options to install OpenVINO Development Tools:

#### Installing Default Components

To install [default components in the package](#whats-in-the-package), use the following command:
```
pip install openvino-dev
```

#### Installing Components for Specific Frameworks

To install and configure the components of the package for working with specific frameworks, use the following command:
```sh
pip install openvino-dev[extras]
```
 where `extras` has the following values: 

| Extras Value                    | DL Framework                                                                     |
| :-------------------------------| :------------------------------------------------------------------------------- |
| caffe                           |   [Caffe*](https://caffe.berkeleyvision.org/)                                    |
| kaldi                           |   [Kaldi*](https://github.com/kaldi-asr/kaldi)                                   |

> | mxnet                           |   [MXNet*](https://mxnet.apache.org/)                                            |

Reviewer: ```suggestion
| mxnet                           |   [Apache MXNet*](https://mxnet.apache.org/)                                            |
```

| onnx                            |   [ONNX*](https://github.com/microsoft/onnxruntime/)                             |
| pytorch                         |   [PyTorch*](https://pytorch.org/)                                               |
| tensorflow                      |   [TensorFlow* 1.x](https://www.tensorflow.org/versions#tensorflow_1)            |
| tensorflow2                     |   [TensorFlow* 2.x](https://www.tensorflow.org/versions#tensorflow_2)            |

For example, to install and configure the components for working with TensorFlow 2.x, MXNet and Caffe, use the following command:  
   ```sh
   pip install openvino-dev[tensorflow2,mxnet,caffe]
   ```

> > **NOTE**: Support of Model Optimizer in TensorFlow 1.x environment is deprecated. Use TensorFlow 2.x environment to convert both TensorFlow 1.x and 2.x models.

Reviewer: ```suggestion
> **NOTE**: Model Optimizer support for TensorFlow 1.x environment has been deprecated. Use TensorFlow 2.x environment to convert both TensorFlow 1.x and 2.x models.
```
